sourcemash/user_items/forms.py

```diff
@@ -0,0 +1,12 @@
+from wtforms import IntegerField
+from wtforms.validators import ValidationError
+
+from flask_wtf import Form
+
+class VoteForm(Form):
+    vote = IntegerField('vote')
+
+    def validate_vote(form, field):
```

Comment:
We don't need to write a validate vote here. Instead, we can use the ```NumberRange(min=None, max=None, message=None)``` validator that comes from wtforms.